### PR TITLE
[PHP][Symfony] Update Controller.mustache

### PR DESCRIPTION
**Pull request by cydrickn**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [ ] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [ ] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [ ] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming langauge.

### Description of the  ###

(details of the change, additional tests that have been done, reference to the issue for tracking, etc)

### CODE REVIEW

**Comment by wing328** [comment]
@cydrickn can you elaborate a bit more on the change? Is that a bug fix or enhancement?

cc @naelrashdeen @ksm2 @jebentier @dkarlovi @mandrean @jfastnacht @ackintosh

**Comment by cydrickn** [comment]
Hi @wing328,

The changes is for php symfony to not response to user the class name and not to response the stack trace of errors for prod environment.
The changes will only response the message of the error if the environment is in production.

Thanks

**Comment by wing328** [comment]
@cydrickn thanks for the explanation. If I might, I would suggest adding an "else" clause:
```php
        if ($this->container->get('kernel')->getEnvironment() === 'prod') {
            return [
                'message'  => $exception->getMessage(),
            ];
        } else {
            return [
                'message'  => $exception->getMessage(),
               'type'     => get_class($exception),
               'previous' => $this->exceptionToArray($exception->getPrevious()),
           ];
        }
```
and also including a 1-liner explaining the difference.

File: modules/swagger-codegen/src/main/resources/php-symfony/Controller.mustache [hunk 1]
```diff
@@ -135,6 +135,12 @@ class Controller
         if (null === $exception) {
             return null;
         }
+        
+        if ($this->container->get('kernel')->getEnvironment() === 'prod') {
```
Comment (dkarlovi) [inline modules/swagger-codegen/src/main/resources/php-symfony/Controller.mustache]: Do not test against this, check for debug mode.

Comment (wing328) [reply modules/swagger-codegen/src/main/resources/php-symfony/Controller.mustache]: Personally, I also prefer the debug flag.

Comment (dkarlovi) [reply modules/swagger-codegen/src/main/resources/php-symfony/Controller.mustache]: It makes sense seeing this is debug information. Production environment can also be run in debug mode.

Comment (cydrickn) [reply modules/swagger-codegen/src/main/resources/php-symfony/Controller.mustache]: Okay i will change this in using debug

Comment (cydrickn) [reply modules/swagger-codegen/src/main/resources/php-symfony/Controller.mustache]: Done changes in 
60a2c7aebf188434ff4713cb4bdce593a1870b8b

**Comment by wing328** [comment]
@cydrickn thanks for the PR, which has been merged into master.

I've updated the Petstore sample for PHP Symfony and removed spaces in the empty line 138 via 8b251555acc7a652e7e6133bbeccd392442466c8

**Pull request merged**